queries.sql

```diff
@@ -1,3 +1,36 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+//Find all animals whose name ends in "mon".
+SELECT * FROM animals
+WHERE name LIKE '%mon';
+
+
+//List the name of all animals born between 2016 and 2019.
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+
+
+//List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT * FROM animals WHERE neutered='True' AND escape_attempts<3;
+
+
+//List the date of birth of all animals named either "Agumon" or "Pikachu".
+SELECT * FROM animals WHERE name='Agumon' OR name='Pikachu';
```

Comment:
- [ ] Here we have to select the `date_of_birth`.
`SELECT date_of_birth FROM animals WHERE name = 'Pikachu' OR name = 'Agumon';`